roles/openshift_logging/files/generate-jks.sh

```diff
@@ -0,0 +1,21 @@
+#! /bin/sh
+
+set -ex
+
+keytool -genkey -alias {{component}} -keystore {{generated_certs_dir}}/{{component}}.jks -keypass kspass -storepass kspass \
+-keyalg RSA -keysize 2048 -validity 712 -dname "CN={{component}}, OU=OpenShift, O=Logging" {{extensions.stdout}}
+
+keytool -certreq -alias {{component}} -keystore {{generated_certs_dir}}/{{component}}.jks -storepass kspass \
+-file {{generated_certs_dir}}/{{component}}-jks.csr -keyalg RSA -dname "CN={{component}}, OU=OpenShift, O=Logging" {{extensions.stdout}}
+
+# this currently fails since we don't provide openssl as part of the ES image...
+openssl ca -in {{generated_certs_dir}}/{{component}}-jks.csr -notext -out {{generated_certs_dir}}/{{component}}-jks.crt \
+-config {{generated_certs_dir}}/signing.conf -extensions v3_req -batch -extensions server_ext
```

Comment:
@ewolinetz  has made attempts to create certs and convert them to jks but had issues with ES being able to start with them (at least i think that was the issue).  We have it on our backlog as targeted 3.5 items to try and generate jks certs using the ES image as a runonce pod.  @ewolinetz has done some initial work towards that goal.